### PR TITLE
DOC: fortran

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #171
Added note about fortran dependency in the installation instructions.

## Type of change
- Documentation update

# How Has This Been Tested?

# Checklist:

- [X] Make sure you are merging into the ``develop`` (not ``main``) branch
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by jklenzing** [comment]
not sure why all the old changes are showing up

**Comment by jklenzing** [comment]
Updating reviewers since changes were co-written

File: docs/installation.rst [hunk 1]
```diff
@@ -32,21 +32,24 @@ or, on windows,
...
 -----------------
 
-By default, sami2py uses gfortran and make to compile the fortran executables.  You can get make through pip
+By default, sami2py uses gfortran and make to compile the fortran executables.
+If you don't have a fortran compiler, gfortran is included as part of the latest
+gcc package.  You can get this from several locations.
+
+For Mac OS X, you can install gcc through package managers such as `brew <https://brew.sh/>`_.
+
+For windows, multiple setup options are discussed at https://www.scivision.dev/cmake-install-windows
```
Comment (aburrell) [inline docs/installation.rst:41]: Do you want to leave a note about making sure the Windows tools are installed before the compiler is?

Comment (jklenzing) [reply docs/installation.rst:41]: Since I don't use windows, a suggested line would be good.

Comment (aburrell) [reply docs/installation.rst:41]: I gave you my best suggested line above (and you know my equal incompetence with Windows).  Basically the problem you helped troubleshoot for AACGMV2 at ISEA.  If you don't want to bother, that's also fine.

Comment (jklenzing) [reply docs/installation.rst]: updated as best as i can

**Review by aburrell — APPROVED** [review]

**Pull request merged**